app/src/pages/inside/dashboardItemPage/modals/widgetWizardModal/widgetWizardModal.jsx

```diff
@@ -24,10 +24,15 @@ const messages = defineMessages({
 
 @withModal('widgetWizardModal')
 @injectIntl
-@connect(null, {
-  destroyWizardForm: () => destroy(WIDGET_WIZARD_FORM),
-  hideModalAction,
-})
+@connect(
+  (state) => ({
+    formValues: getFormValues(WIDGET_WIZARD_FORM)(state),
```

Comment:
I suggest using `isDirty` selector to prevent rerendering on every field change.